pages/signup.tsx

```diff
@@ -39,59 +65,71 @@ const SignUp: NextPage = () => {
 							}}
 							className='mt-4 flex flex-col space-y-4'
 						>
-							<div className='flex w-[90%] flex-col space-y-2'>
-								<p>Email</p>
+							<div className='flex w-full flex-col space-y-2'>
+								<label htmlFor='email-address' className='text-sm text-gray-600'>
+									Email address
+								</label>
 								<input
 									type='text'
+									id='email-address'
 									placeholder='example@gmail.com'
 									required
-									className='input-field outline-none'
+									className='relative block w-full appearance-none rounded-md border border-gray-300 px-3 py-2 text-gray-900 placeholder-gray-500 focus:z-10 focus:border-indigo-500 focus:outline-none focus:ring-indigo-500 sm:text-sm'
 									value={email}
 									onChange={e => setEmail(e.target.value)}
 								/>
 							</div>
-							<div className='flex w-[90%] flex-col space-y-2'>
-								<p>Username</p>
+							<div className='flex w-full flex-col space-y-2'>
+								<label htmlFor='username' className='text-sm text-gray-600'>
+									Username
+								</label>
 								<input
 									type='text'
+									id='username'
 									placeholder='example'
 									required
-									className='input-field outline-none'
+									className='relative block w-full appearance-none rounded-md border border-gray-300 px-3 py-2 text-gray-900 placeholder-gray-500 focus:z-10 focus:border-indigo-500 focus:outline-none focus:ring-indigo-500 sm:text-sm'
 									value={username}
 									onChange={e => setUsername(e.target.value)}
 								/>
 							</div>
-							<div className='flex w-[90%] flex-col space-y-2 '>
-								<p>Password</p>
+							<div className='flex w-full flex-col space-y-2 '>
+								<label htmlFor='password' className='text-sm text-gray-600'>
+									Password
+								</label>
 								<input
 									type='password'
+									id='password'
 									required
-									className='input-field outline-none'
+									className='relative block w-full appearance-none rounded-md border border-gray-300 px-3 py-2 text-gray-900 placeholder-gray-500 focus:z-10 focus:border-indigo-500 focus:outline-none focus:ring-indigo-500 sm:text-sm'
 									placeholder='*****'
 									value={password}
 									onChange={e => setPassowrd(e.target.value)}
 								/>
 							</div>
-							<div className='flex w-[90%] flex-col space-y-2 '>
-								<p>Confirm Password</p>
+							<div className='flex w-full flex-col space-y-2 '>
+								<label htmlFor='confirm-password' className='text-sm text-gray-600'>
+									Confirm Password
```

Comment:
what is this for?

Reply:
just to confirm password before signup. it was in the former signup form so I added it to the new one also

Reply:
should I remove it?

Reply:
I suggest keeping the `Confirm Password` input as it forces users to double-check their password before signing up. Can you add this back?

Reply:
@iShibi is there any modern website that uses that field? I just haven't really seen it recently.

Reply:
Plus as long as the email is correct, the user can always reset their password. So if anything, there should be a "confirm email" field, but I don't think that's necessary, either.

Reply:
That's a valid point. I guess we are good without it.